packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -0,0 +1,262 @@
+// @ts-nocheck
```

Comment:
why is this whole file marked as new and not detected as renamed like all the others? this means we will lose all the history of changes for this file, doesn't it?

Reply:
Fixed.